work-scheduler-web/src/main/webapp/WEB-INF/jsp/registration.jsp

```diff
@@ -5,29 +5,34 @@
   Time: 19:48
   To change this template use File | Settings | File Templates.
 --%>
-<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ page   contentType="text/html;charset=UTF-8" language="java" %>
+<%@ taglib prefix="form"    uri="http://www.springframework.org/tags/form" %>
+
 <html>
     <head>
-        <title>Title</title>
+        <title>Registration</title>
     </head>
     <body>
-        <form method="post" action="${pageContext.request.contextPath}/registration">
-            <label for="personal-name">Employee name:</label>
-            <input type="text" name="name" id="personal-name"/>
-
-            <label for="birth-date">Birth date:</label>
-            <input type="date" name="date" id="birth-date"/>
+        <main>
+            <form:form modelAttribute="employee" action="${pageContext.request.contextPath}/registration">
+                <div>
+                    <form:label path="name">Name: </form:label>
+                    <form:input required="required" path="name"/>
+                </div>
 
-            <label for="personal-id">Personal ID:</label>
-            <input type="text" name="ID" id="personal-id">
+                <div>
+                    <form:label path="birthday">Birthday: </form:label>
+                    <form:input required="required" path="birthday" type="date"/>
+                </div>
 
-            <label for="medical-care">Medical care:</label>
-            <input type="text" name="medical-care" id="medical-care">
+                <div>
+                    <form:label path="email">Mail: </form:label>
+                    <form:input required="required" path="email"/>
+                </div>
 
-            <label for="personal-mail">Personal mail:</label>
-            <input type="email" name="mail" id="personal-mail">
+                <form:button>Submit</form:button>
 
-            <button type="submit">Register</button>
-        </form>
+            </form:form>
+        </main>
     </body>
 </html>
```

Comment:
Allow the other 2 employee fields to be specified: Personal ID card number and medical care ID.